src/sass/app/trade.scss

```diff
@@ -1062,3 +1062,85 @@
         }
     }
 }
+
+#deriv_banner_container {
+    display: flex;
+    width: 100%;
+    height: 200px;
+    position: relative;
+    margin-bottom: 40px;
+
+    .deriv_banner_background {
+        object-fit: fill;
+        width: 100%;
+        height: 200px;
+        position: absolute;
+    }
+    .platform_image {
+        max-height: 157px;
+        max-width: 245px;
+        margin-top: 22px;
+        margin-left: 16px;
+        z-index: 10;
+
+        @media (max-width: 768px) {
+            display: none;
+        }
+    }
+    .watermark_background {
+        position: absolute;
+        right: 12.3%;
+        top: 19.5%;
+
+        @media (max-width: 425px) {
+            transform: scale(0.7, 1);
+            right: -10%;
+            width: 450px;
+        }
+        @media (max-width: 325px) {
+            transform: scale(0.7, 1);
+            right: -20%;
+            width: 450px;
+        }
+    }
+    div {
+        margin: 49px 0 10px 16px;
+        z-index: 1;
+
+        h2, h5 {
+            color: $COLOR_WHITE;
+        }
+        h2 {
+            font-weight: bold;
+            margin-bottom: 10px;
+
+            @media (max-width: 768px) {
+                font-size: $FONT_SIZE_N;
+            }
+        }
+        h5 {
+            @media (max-width: 768px) {
+                font-size: $FONT_SIZE_S;
+            }
+        }
+    }
+    div.button_container {
+        margin: 30px 0 0;
+
+        a.deriv_button {
+            background-color: $COLOR_SHARP_RED;
+            width: 123px;
+            height: 40px;
+            color: $COLOR_WHITE;
+            padding: 10px 16px;
+            font-size: $FONT_SIZE_S;
+            font-weight: bold;
+            text-decoration: none;
+            border-radius: 4px;
+        }
+    }
+
+    @media (max-width: 1024px) {
```

Comment:
@arash-fs from testing deriv.app, it remains visible until mobile size `767`, not tablet size